cmd/emp/hkclient/creds.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 type NetRc struct {
-	netrc.Netrc
```

Comment:
Why are we wrapping Netrc in our own struct?

Reply:
I have no idea.  I only modified this to quiet the warning, I don't want to visit it as an architectural decision.

Reply:
I don't either; I was just hoping you had some knowledge I could absorb...